src/sentry/migrations/0005_fix_content_types.py

```diff
@@ -47,4 +47,4 @@ class Migration(migrations.Migration):
         )
     ]
 
-    operations = [migrations.RunPython(fix_content_types)]
+    operations = [migrations.RunPython(fix_content_types, migrations.RunPython.noop)]
```

Comment:
currently django would raise exception if we tried to reverse the migrations because the data migration has no backwards. This avoids that problem